test_runner/fixtures/safekeeper/http.py

```diff
@@ -161,6 +162,16 @@ def timeline_status(
             walreceivers=walreceivers,
         )
 
+    # Get timeline_start_lsn, waiting until it's nonzero. It is a way to ensure
+    # that the timeline is fully initialized at the safekeeper.
+    def get_non_zero_timeline_start_lsn(self, tenant_id: TenantId, timeline_id: TimelineId) -> Lsn:
+        def timeline_start_lsn_non_zero():
+            s = self.timeline_status(tenant_id, timeline_id).timeline_start_lsn
+            assert s > Lsn(0)
+
+        wait_until(30, 1, timeline_start_lsn_non_zero)
+        return self.timeline_status(tenant_id, timeline_id).timeline_start_lsn
```

Comment:
```suggestion
            assert s > Lsn(0)

        wait_until(30, 1, timeline_start_lsn_non_zero)
        return self.timeline_status(tenant_id, timeline_id).timeline_start_lsn
```
```suggestion
            assert s > Lsn(0)
            return s

        return wait_until(30, 1, timeline_start_lsn_non_zero)
```

Reply:
applied

Reply:
aah yes sorry I forgot the return type. always something with python :)